### ISSUE TITLE
hw01 ready for grading

### ISSUE THREAD

**Issue by sepkamal** [sepkamal/STAT545-hw-Kamal-Sepehr#1]
Hi @vincenzocoia @gvdr @ksedivyhaley @joeybernhardt @mynamedaike @pgonzaleze @derekcho

Please see the link below for the rendered .md file. The README.md file and both .Rmd, and .md files on gapminder for homework 1 are all in this repo.

https://github.com/sepkamal/STAT545-hw01-kamal-sepehr/blob/master/hw01_gapminder.md

Thanks,
Sep

**Comment by HScheiber** [comment]
Awesome! Nice README.md file which gives a good introduction and demonstrates lots of aspects of markdown (linking, pictures, headers, bold, strike through, table, emojis). Readme file links directly to the gap-minder exploration file.

Clearly went ahead put more work than was required into the gap-minder exploration plots! However, the gapminder exploration file has no explanation text, which makes it hard to follow. My suggestion is just to add text explaining what your plots mean. You can even add plot titles! 

Readme file includes good description of how changes to the files were done.

Overall I give it a check plus.

**Comment by Maria815** [comment]
Good! Nice README.md file which gives a good introduction and demonstrates lots of aspects of markdown. I would suggest adding quotation or checkbox to enrich your introduction.
Complete gap-minder exploration file and description of how changes were done nicely. 
Overall I give a check plus.

**Comment by JoeyBernhardt** [comment]
Correctly named issue: Yes
Link to work to be reviewed: in this issue
Use 4 or more aspects of markdown: Yes (headings, list, formatting, images)
Reflections / how it was done: Yes
Gapminder exploration: Yes

Comments:

Very nicely done! Nice use of images, links etc! Reflections on the process were nicely detailed as well. Gapminder exploration was good, although next time, I recommend adding a little narrative about the figures are saying -- is there anything surprising in there? What story do they tell?

Note: your mark will be sent out later.
